lib/upipe-framers/upipe_s337_framer.c

```diff
@@ -410,7 +410,7 @@ static int upipe_s337f_set_flow_def(struct upipe *upipe, struct uref *flow_def)
         return UBASE_ERR_ALLOC;
     }
 
-    flow_def = upipe_s337f_store_flow_def_input(upipe, flow_def_dup);
+    upipe_s337f_store_flow_def_input(upipe, flow_def_dup);
```

Comment:
You need to do something with the built flow def otherwise it's a memleak. Usually you store_flow_def() it.